saleor/graphql/account/schema.py

```diff
@@ -0,0 +1,61 @@
+import graphene
+from graphql_jwt.decorators import permission_required
+
+from ..core.fields import PrefetchingConnectionField
+from ..descriptions import DESCRIPTIONS
+from .mutations import (
+    AddressCreate, AddressDelete, AddressUpdate, CustomerCreate,
+    CustomerDelete, CustomerPasswordReset, CustomerRegister, CustomerUpdate,
+    PasswordReset, SetPassword, StaffCreate, StaffDelete, StaffUpdate)
+from .resolvers import (
+    resolve_address_validator, resolve_customers, resolve_staff_users)
+from .types import AddressValidationData, AddressValidationInput, User
+
+
+class AccountQueries(graphene.ObjectType):
+    address_validator = graphene.Field(
+        AddressValidationData,
+        input=graphene.Argument(AddressValidationInput, required=True))
+    customers = PrefetchingConnectionField(
+        User, description='List of the shop\'s users.',
```

Comment:
> users

It would probably be better to say `customers` so stay consisting with the name.